package.json

```diff
@@ -1,16 +1,27 @@
 {
     "private": true,
     "devDependencies": {
-        "@babel/cli": "^7.23.0",
-        "@babel/core": "^7.23.0",
-        "@babel/plugin-transform-modules-commonjs": "^7.23.0",
+        "@babel/cli": "^7.28.3",
+        "@babel/core": "^7.28.4",
+        "@babel/plugin-transform-modules-commonjs": "^7.27.1",
         "@types/node": "18.19.54",
         "chalk": "5.1.1",
         "inline-fixtures": "1.1.0",
         "jsonpath-plus": "7.2.0",
         "typescript": "^5.8.0"
     },
     "pnpm": {
+        "onlyBuiltDependencies": [
+            "@aspect-test/c",
+            "@figma/nodegit",
```

Comment:
@dzbarsky I added a few things here since you approved the PR.

Adding `fsevents` reduced the diff on the snapshot for that file. Adding `@kubernetes/client-node` (which I think I did before you approved) is what caused the `@kubernetes/client-node` `packageExtensions` entry down below due to pnpm9 deciding transitive deps differently and causing a tsc error with a newer version of `@types/node`.